scripts/prepare_network.py

```diff
@@ -180,13 +180,9 @@ def average_every_nhours(n, offset):
 if __name__ == "__main__":
     # Detect running outside of snakemake and mock snakemake for testing
```

Comment:
Delete? Small thing, but this is not yet consistent.

Reply:
yep